grammar.js

```diff
@@ -1936,6 +1938,19 @@ module.exports = grammar({
 
     macro_verbatim: $ => seq('verbatim', 'do'),
 
+    array_like_type_literal: $ => seq(
+      field('receiver', choice($.constant, $.generic_instance_type)),
+      field('arguments', $.tuple),
```

Comment:
Nitpick: I'm not quite sure about these field names, maybe `name` and `values`?

Reply:
Was trying to match the same as `call`, but that works for me

Reply:
I may have been influenced by the fact that the parser represents this as a `HashLiteral` with a `name` property